check_online_presence.py

```diff
@@ -135,30 +181,35 @@ def main():
 
     sites_to_check = find_sites_to_check(args, data)
 
+    # Start threads
+    threads = list()
     for site in sites_to_check:
         if not site['valid']:
             warn("[!] Skipping %s - Marked as not valid." % site['name'])
             continue
 
         if args.username:
-            check_site(site, args.username,
-                       if_found     = lambda url: positive("[+] User found at %s" % url),
-                       if_not_found = lambda url: neutral( "[-] User not found at %s" % url),
-                       if_neither   = lambda url: error(   "[!] Error. The check implementation is broken for %s" % url))
+            run_type = 'username'
```

Comment:
Extracting the `message_to_print` method makes the code less readable.
And here we are dealing with potentially non-professional developers, so the readability of the code is a key factor.
Is there a better way to structure that?

Reply:
I will look into it

Reply:
Hey @salaheldinaz any progress on this? Not pushing...just checking.

Reply:
> Hey @salaheldinaz any progress on this? Not pushing...just checking.
I missed this notification.
I will continue work on it  👍